dropbox/oauth.py

```diff
@@ -163,6 +189,11 @@ def _finish(self, code, redirect_uri):
         else:
             expires_in = None
 
+        if 'scope' in d:
+            scope_list = d
```

Comment:
scope_list = d["scope"]

Reply:
This reveals that we might need some test coverage on `finish` method.

Reply:
Good catch, changed and can add some coverage